src/relay/api/resources.py

```diff
@@ -341,7 +341,7 @@ def get(self, args, network_address: str, user_address: str):
             )
         except TimeoutException:
```

Comment:
We can probably remove the time out stuff, as this should be related to the old contract code

Reply:
I think it is true for all of them except this one: https://github.com/trustlines-protocol/relay/pull/497/files#diff-a005b72d629c8034ee162be425c13202R232
It spawns greenlet for each exchange addresses and make a query to an ethindex_db instance for each of them (one db instance by addresses), so it can time out.